src/test/java/com/egov/core-services/pretests/updatePasswordNoLoginPretest.feature

```diff
@@ -0,0 +1,134 @@
+Feature: Update user profile
+
+
```

Comment:
Keep always uniform 1 line if you feel like to differentiate sections but not 2 or more lines. Please delete 1 empty line.